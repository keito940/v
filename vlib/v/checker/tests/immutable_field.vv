struct A {
pub:
	i1 int
}

fn main() {
	a := A{1}
	a.i1 = 2
}
