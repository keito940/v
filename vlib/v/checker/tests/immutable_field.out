vlib/v/checker/tests/immutable_field.v:8:4: error: field `i1` of struct `A` is immutable
    6 | fn main() {
    7 |     a := A{1}
    8 |     a.i1 = 2
      |       ~~
    9 | }
